scripts/macOS/package-au.sh

```diff
@@ -37,7 +37,9 @@ cp -R "$BUNDLE_RES_SRC_LOCATION" "$BUNDLE_DIR/Contents/Resources"
 if [[ -z "$SURGE_USE_VECTOR_SKIN" ]]; then
     cp $BITMAP_SRC_LOCATION/* "$BUNDLE_DIR/Contents/Resources/"
 else
-    rm "$BUNDLE_DIR/Contents/Resources/bmp?????.png"
+    rm "$BUNDLE_DIR/Contents/Resources/bmp*.png"
```

Comment:
What is this change BTW?

Reply:
That change makes sure you blow away bitmaps which also have form 00100@2x.png names